### PR TITLE
Converted to use rally-sbt-plugin

### PR DESCRIPTION
**Pull request by jeffmay**
@htmldoug @slustbader

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -5,8 +5,6 @@ organization := "com.rallyhealth"
 
 organizationName := "Rally Health"
 
-version := "1.0.0"
-
 crossScalaVersions := Seq("2.11.6", "2.10.4")
 
 scalacOptions := {
```
Comment (slustbader) [inline build.sbt]: I think you can punt this too.

File: build.sbt [hunk 2]
```diff
@@ -38,18 +36,3 @@ publishArtifact in (Compile, packageDoc) := false
```
Comment (slustbader) [inline build.sbt]: and ```publishArtifact in (Compile, packageDoc) := false```

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 // See https://wiki.audaxhealth.com/display/ENG/Build+Structure#BuildStructure-Localconfiguration
 credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
 
-resolvers += "Rally Plugin Releases" at "https://artifacts.werally.in/artifactory/plugins-release"
+resolvers += Resolver.url("Rally Plugin Releases", url("https://artifacts.werally.in/artifactory/ivy-plugins-release"))(Resolver.ivyStylePatterns)
+
+addSbtPlugin("com.rallyhealth" %% "rally-sbt-plugin" % "0.0.2")
```
Comment (slustbader) [inline project/plugins.sbt]: Latest is 0.0.3

File: project/plugins.sbt [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
 
-addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "0.2.0")
```
Comment (slustbader) [inline project/plugins.sbt]: FAIL: you still need the rally-versioning plugin, and it has to be listed before the rally-sbt-plugin.

Comment (htmldoug) [inline project/plugins.sbt]: Either that or you'll need to set and update the version manually.

Comment (jeffmay) [inline project/plugins.sbt]: Oh, I thought that was part of the `rally-sbt-plugin`. Will fix.

Question of preference: There is a community version of this that has fixed versions. If we use the community fixed version, then we can have some parody with the community maintained repo. Do you think it is better to do that, or just have SHA1 commit history parody with merges and keep the listed version number separate?

File: build.sbt (hunk 1 above)
Comment (jeffmay) [inline build.sbt]: I was leaving this to keep better parody with the community version.

File: project/plugins.sbt (hunk 4 above)
Comment (slustbader) [inline project/plugins.sbt]: I think manual versions are probably OK here, since this shouldn't change frequently.

(Also, I assume you mean parity.)

Comment (htmldoug) [inline project/plugins.sbt]: @rcmurphy drew influences from it when he wrote our version. There were things it didn't do that he wanted.

File: project/plugins.sbt (hunk 3 above)
Comment (htmldoug) [inline project/plugins.sbt]: you'll want the latest.

File: project/plugins.sbt (hunk 4 above)
Comment (rcmurphy) [inline project/plugins.sbt]: @jeffmay I wanted to be able to take a given commit and 'release' it without having to go through our entire PR process just to bump the version number.

Comment (jeffmay) [inline project/plugins.sbt]: And by "parody" I mean "derpity".

Ok, I think I'm gonna keep some version parity while the two projects only differ in build.sbt settings but are identical in code.

**Comment by egprentice** [comment]
LGTM

File: project/plugins.sbt [hunk 5]
```diff
@@ -1,6 +1,7 @@
 // See https://wiki.audaxhealth.com/display/ENG/Build+Structure#BuildStructure-Localconfiguration
 credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
 
-resolvers += "Rally Plugin Releases" at "https://artifacts.werally.in/artifactory/plugins-release"
+resolvers += Resolver.url("Rally Plugin Releases", url("https://artifacts.werally.in/artifactory/ivy-plugins-release"))(Resolver.ivyStylePatterns)
+
+addSbtPlugin("com.rallyhealth" %% "rally-sbt-plugin" % "0.0.3")
```
Comment (jeffmay) [inline project/plugins.sbt]: Unfortunately, this PR took so long that the version is now 0.1.0

**Pull request merged**